.github/scripts/gradlew_recursive.sh

```diff
@@ -18,17 +18,19 @@ set -xe
 
 # Default Gradle settings are not optimal for Android builds, override them
 # here to make the most out of the GitHub Actions build servers
-GRADLE_OPTS="$GRADLE_OPTS -Xms4g -Xmx4g"
-GRADLE_OPTS="$GRADLE_OPTS -XX:MaxMetaspaceSize=512m"
+GRADLE_OPTS="$GRADLE_OPTS -Xmx5120m"
 GRADLE_OPTS="$GRADLE_OPTS -XX:+HeapDumpOnOutOfMemoryError"
-GRADLE_OPTS="$GRADLE_OPTS -Dfile.encoding=UTF-8"
+GRADLE_OPTS="$GRADLE_OPTS -Dorg.gradle.daemon=false"
+GRADLE_OPTS="$GRADLE_OPTS -Dorg.gradle.workers.max=2"
+GRADLE_OPTS="$GRADLE_OPTS -Dkotlin.incremental=false"
+GRADLE_OPTS="$GRADLE_OPTS -Dkotlin.compiler.execution.strategy=in-process"
```

Comment:
Would you recommend these changes to all samples, or are they specific to this repo?

Reply:
Yep, I've been using them in Tivi for ~6 months and it's been rock solid: https://github.com/chrisbanes/tivi